packages.yml

```diff
@@ -1,3 +1,4 @@
 packages:
-  - package: fivetran/pinterest_source
-    version: [">=0.7.0", "<0.8.0"]
\ No newline at end of file
+    - git: https://github.com/fivetran/dbt_pinterest_source.git
+      revision: feature/pinterest_v5_API_support
+      warn-unpinned: false
```

Comment:
Reminder to switch before merge and release.

Reply:
@fivetran-reneeli in addition to updating this before merging I realized the README has not been updated with this package new version as well as the dependency matrix needs to be updated to point to the new source package range. Please update those before merging or kicking off the release process.